bundles/binding/org.openhab.binding.mystromecopower/src/main/java/org/openhab/binding/mystromecopower/internal/MyStromEcoPowerBinding.java

```diff
@@ -172,7 +180,7 @@ protected void execute() {
                         }
                     }
                 } else {
-                    logger.warn("The device itemName '{}' not found on discovery. Verify device is not offline",
+                    logger.warn("The device itemName '{}' not found on discovery verify device is not offline",
```

Comment:
If I merge this as is, could you then open a new PR to revert this change in the message?  Otherwise, could you revert the change with a new commit on this PR branch, and then squash so there is one commit?